app/weather_api/services/met_service.py

```diff
@@ -0,0 +1,44 @@
+import asyncio
+
+import aiohttp
+import requests
```

Comment:
It seems like requests is not used anymore, if that's the case you should remove this import